docs/dev/connectors/file_sink.md

```diff
@@ -0,0 +1,808 @@
+---
+title: "File Sink"
+nav-title: File Sink
+nav-parent_id: connectors
+nav-pos: 5
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+* This will be replaced by the TOC
+{:toc}
+
+This connector provides a unified Sink for `BATCH` and `STREAMING` that writes partitioned files to filesystems
+supported by the [Flink `FileSystem` abstraction]({{ site.baseurl}}/ops/filesystems/index.html). It provides the
+same guarantees for both `BATCH` and `STREAMING` execution by exposing the same API while having different runtime 
```

Comment:
It's not the `FileSink` that does this. The Flink runtime is responsible for this.

Reply:
True, I will remove this.